evap/staff/importers.py

```diff
@@ -498,25 +498,33 @@ def write_enrollments_to_db(self, semester, vote_start_datetime, vote_end_date):
             for evaluation, participants in participants_per_evaluation.items():
                 evaluation.participants.add(*participants)
 
-        msg = format_html(
-            _("Successfully created {} courses/evaluations, {} participants and {} contributors:"),
-            len(self.evaluations),
-            len(students_created),
-            len(responsibles_created),
-        )
-        msg += create_user_list_html_string_for_message(students_created + responsibles_created)
+            msg = format_html(ngettext('Successfully created one course/evaluation,', 'Successfully created {evaluation} courses/evaluations,', len(self.evaluations),).format(evaluation = len(self.evaluations)))
+            if len(students_created) == 0 and len(responsibles_created) == 0:
+                msg += format_html(_(" no students and no contributors."))
+            else:
+                if len(students_created) == 0:
+                    msg += format_html(_(" no students"))
+                else:
+                    msg += format_html(ngettext(' one student', ' {users} students', len(students_created),).format(users = len(students_created)))
+                    if len(responsibles_created) == 0:
+                        msg += format_html(_(" and no contributors:"))
+                    else:
+                        msg += format_html(ngettext(' and one contributor:', ' and {users} contributors:', len(responsibles_created),).format(users = len(responsibles_created)))
+                msg += create_user_list_html_string_for_message(students_created + responsibles_created)
+
```

Comment:
Maybe we should try to prevent these 3 levels of nested if-else blocks?
We could try something like:
```python3
student_count = len(students_created)
student_word = _("no") if student_count == 0 else str(student_count)  # TODO: Should use pgettext
responsible_count = len(responsibles_created)
responsible_word = _("no") if responsible_count == 0 else str(responsible_count)  # TODO: Should use pgettext

student_phrase = ngettext("{count} student", "{count} students", student_count).format(count=student_word)
responsible_phrase = ngettext("{count} responsible", "{count} responsibles", responsible_count).format(count=responsible_word)

message = ngettext(
    'Successfully created one course/evaluation, {student_phrase} and {reponsible_phrase}',
    'Successfully created {evaluation} courses/evaluations, {student_phrase} and {reponsible_phrase}''
).format(responsible_phrase=responsible_phrase, student_phrase=student_phrase)
```

what do you think?